dashboard/app/dsl/script_dsl.rb

```diff
@@ -121,4 +121,72 @@ def i18n_strings
   def self.parse_file(filename)
     super(filename, File.basename(filename, '.script'))
   end
+
+  def self.serialize(script, filename)
+    s = []
+
+    # Legacy script IDs
+    legacy_script_ids = {
+      :'20-hour' => 1,
+      :'Hour of Code' => 2,
+      :'edit-code' => 3,
+      events: 4,
+      flappy: 6,
+      jigsaw: 7,
+    }.with_indifferent_access
+    s << "id '#{legacy_script_ids[script.name]}'" if legacy_script_ids[script.name]
+
+    s << "professional_learning_course '#{script.professional_learning_course}'" if script.professional_learning_course
+    s << "peer_reviews_to_complete #{script.peer_reviews_to_complete}" if script.peer_reviews_to_complete
+
+    s << "hidden 'false'" unless script.hidden
+    s << "login_required 'true'" if script.login_required
+    s << 'hideable_stages true' if script.hideable_stages
```

Comment:
is this intentional for hideable_stages to be a `true` boolean argument while the others are `'true'`/`'false'` string arguments?